lib/utils/addon-utils.ts

```diff
@@ -12,41 +13,44 @@ export function getAddOnNameOrId(addOn: ClusterAddOn): string {
   return addOn.id ?? addOn.constructor.name;
 }
 
-export function isOrderedAddOn(addOn: ClusterAddOn) : boolean {
-    return Reflect.getMetadata("ordered", addOn.constructor) ?? Reflect.getMetadata("ordered", addOn) ?? false;
+export function isOrderedAddOn(addOn: ClusterAddOn): boolean {
```

Comment:
let's format only modified lines to make reviews easier. 
if a file is not formatted, create a separate PR to reformat.

Reply:
Acknowledge.

will remove from the PR.